test/assignment-tests.js

```diff
@@ -0,0 +1,80 @@
+import test from 'tape';
+import compose from '../examples/compose';
+
+const assignmentProps = [
+  'methods',
+  'properties',
+  'staticProperties',
+  'deepStaticProperties',
+  'propertyDescriptors',
+  'staticPropertyDescriptors'
+];
+
+const build = (num) => {
+  const composable = {};
+
+  assignmentProps.forEach(prop => {
+    composable[prop] = {
+      [num]: num,
+      override: num
+    };
+  });
+
+  return { compose: composable };
```

Comment:
Yeah, I don't think it's a problem, but I should clean it up and either pass a POJO or a real stamp, depending on the test. =)